app/controllers/package.scala

```diff
@@ -63,6 +68,31 @@ package object controllers {
 
   implicit class SettableOpsRunner[A](userAnswersWriter: UserAnswersWriter[Write[A]]) {
 
+    def removeOffices(previousSelectedCountry: Option[Country], selectedCountry: Country): UserAnswersWriter[Write[A]] =
+      userAnswersWriter.flatMapF {
+        case (page, userAnswers) =>
+          previousSelectedCountry match {
+            case Some(previousCountry) =>
+              Right(
+                (page,
+                 userAnswers
+                   .findAndRemoveOffices(OfficesOfTransitSection,
+                                         OfficeOfTransitSection,
+                                         OfficeOfTransitPage,
+                                         previousCountry.code.code,
+                                         selectedCountry.code.code
+                   )
+                   .findAndRemoveOffices(OfficesOfExitSection, OfficeOfExitSection, OfficeOfExitPage, previousCountry.code.code, selectedCountry.code.code)
+                )
```

Comment:
Again, me being OCD haha, can we format these two `findAndRemoveOffices` calls the same way